scanner/src/routes/style.css

```diff
@@ -15,3 +15,13 @@ div:has(> video) {
 video {
   object-fit: cover;
 }
+
+.background {
+  background: rgba(16, 25, 32, 1);
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 0;
+}
```

Comment:
Why do we want a floating background instead of having a background in the `body`?